department-of-veterans-affairs/va.gov-cms
# VACMS-3674: Add new field to R&S terms.

## Description
See #3674 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/101906222-9cf6ab80-3b86-11eb-83b6-1a15b590f3a1.png)
![image](https://user-images.githubusercontent.com/2404547/101906228-a08a3280-3b86-11eb-94bf-8a2290e7b95c.png)

## QA steps
- As an administrator, go to `/taxonomy/term/308/edit` & `/taxonomy/term/268/edit`
- Visually verify that the new field appears below the `Published` boolean

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
